frontend/src/components/organisms/form-sections/OrganisationSection/TechnicalTeamCard.js

```diff
@@ -1,22 +1,63 @@
-import React, { useContext, useMemo } from 'react';
+import React, { useContext, useEffect, useMemo } from 'react';
 import { Card } from '../../../molecules/Card';
-import SelectInput from '../../../atoms/inputs/SelectInput';
 import { FormContext } from '../../../templates/Form';
+import { TextInputWithSuggestions } from '../../../molecules/TextInputWithSuggestions';
+import RadioInput from '../../../atoms/inputs/RadioInput';
+import Button from '../../../atoms/hyperTexts/Button';
 import TextInput from '../../../atoms/inputs/TextInput';
-import TextInputWithSuggestions from '../../../molecules/TextInputWithSuggestions';
+import Alert from '../../../atoms/Alert';
+import Link from '../../../atoms/hyperTexts/Link';
+import { HideSectionsContext } from '../../../templates/Form/HideSectionsContainer';
+
+const typeOptions = [
+  { id: 'software_company', label: 'Votre éditeur de logiciel' },
+  { id: 'internal_team', label: 'Votre équipe de développeurs' },
+  { id: 'other', label: 'Autre' },
+];
 
 export const TechnicalTeamCard = ({ editorList = [] }) => {
   const {
     disabled,
     onChange,
+    isUserEnrollmentLoading,
     enrollment: { technical_team_type, technical_team_value },
   } = useContext(FormContext);
 
+  const { setReadyForNextSteps } = useContext(HideSectionsContext);
+
+  const areInputValid = useMemo(() => {
+    if (!technical_team_type) {
+      return false;
+    }
+    if (technical_team_type === 'internal_team') {
+      return true;
+    }
+    if (technical_team_value) {
+      return true;
+    }
+    return false;
+  }, [technical_team_type, technical_team_value]);
+
+  useEffect(() => {
```

Comment:
This logic is a state update depending on a form event. It should be handled in a callback called by the Form on form state change.
I see that it is not available yet in the form hook used here si this is not in the scope of this PR.